js/data.js

```diff
@@ -43,11 +43,14 @@ function createArtPieces(event) {
   var newArtTitleYear = document.createElement('p');
   var newItalics = document.createElement('i');
   var addArtIcon = document.createElement('img');
+  var newArtDescription = document.createElement('p');
```

Comment:
Everything not related to your data model or localStorage should be in `main.js`, not `data.js`